ansible/roles/dashmate/tasks/main.yml

```diff
@@ -30,6 +30,9 @@
     mode: "0750"
   loop:
     - '{{ dashmate_config_dir }}/ssl/{{ dash_network_name }}'
+    - '{{ dashmate_config_dir }}/{{ dash_network_name }}'
+    - '{{ dashmate_config_dir }}/{{ dash_network_name }}/core'
+    - '{{ dashmate_config_dir }}/{{ dash_network_name }}/platform'
 
```

Comment:
Why do we need this?

Reply:
We don't need core and platform for sure, but we do need dashmate config and ssl directories as they are used in next steps before actually starting dashmate node